frontend/src/layout/navigation/OrganizationSwitcher.tsx

```diff
@@ -14,9 +15,9 @@ import { navigationLogic } from './navigationLogic'
 
 export function AccessLevelIndicator({ organization }: { organization: OrganizationBasicType }): JSX.Element {
     return (
-        <div className="AccessLevelIndicator" title={`Your ${organization.name} organization access level`}>
-            {organization.membership_level ? membershipLevelToName.get(organization.membership_level) : '?'}
-        </div>
+        <LemonTag className="AccessLevelIndicator" title={`Your ${organization.name} organization access level`}>
+            {(organization.membership_level ? membershipLevelToName.get(organization.membership_level) : null) || '?'}
```

Comment:
Context for the change? Unrelated to groups.

Reply:
Change is we now use this `LemonTag` component for standardization purposes.